dicom_util/src/main/java/com/google/cloud/healthcare/DicomWebClient.java

```diff
@@ -56,14 +51,15 @@ public DicomWebClient(
   /**
    * Makes a WADO-RS call and returns the multipart response.
```

Comment:
comment is out of date: not a multipart response any more

Reply:
Ok, changed.